test/lib/processors/jsdoc/jsdocGenerator.js

```diff
@@ -26,7 +26,7 @@ test("generateJsdocConfig", async (t) => {
 		.replace(backslashRegex, "\\\\");
 	const destinationPath = path.join("/", "some", "tm\\p", "path")
 		.replace(backslashRegex, "\\\\");
-	const jsapiFilePath = path.join("/", "some", "target", "path", "libraries", "some.namespace.js")
+	const jsapiFilePath = path.join("/", "some", "target", "path", "libraries", "some.projectName.js")
```

Comment:
was it only a consequence of the change in line 13 that the name of the file changed or is i a conscious decision that it should be named after the project?

In Maven, the name was derived from the library name (see the deletion in https://git.wdf.sap.corp:8080/#/c/sapui5/sapui5.runtime/+/4441367/2/src/framework/maven/poms/library-parent/pom.xml ). 

BTW: in that change we abandoned the generation of the jsapi file as Eclipse is no longer supported since then.

Reply:
> was it only a consequence of the change in line 13 that the name of the file changed

Yes, in the test, the project name now differs from the namespace (which can happen for some projects). We decided to use the projectName for the jsapi file. IIRC we preferred it for that since it is guaranteed to be unique per build, while the namespace may not.

> in that change we abandoned the generation of the jsapi file as Eclipse is no longer supported since then.

Should we plan that too for UI5 Tooling 3.0?

Reply:
This mismatch with the Maven build still puzzles me, but okay...

Yes, we should plan the removal of Eclipse code-completion artefacts for 3.0.

Reply:
I don't mind to use the `namespace` for the jsapi file if this ensures an alignment with Maven.

Reply:
I clarified with Merlin that we will not change the file name. We don't expect that it is used anyways and until now nobody has complained about it.

Reply:
Plus, we plan to remove it anyways...

Reply:
Okay with me.